cmd/apply.go

```diff
@@ -208,6 +208,24 @@ func runApply(options *cli.ApplyOptions) error {
 		crdStateStore.Commit(newCluster)
 	}
 
+	if newCluster.CloudManagerSecret != nil {
+		// -------------------------------------------------------------------------------------------------------------
+		//
+		// Here is where we hook in for the new controller logic
+		// This is exclusive to profiles that have a cloud-manager secret defined
+		//
+		logger.Info("Deploying cloud manager secret: %s", newCluster.CloudManagerSecret.ObjectMeta.Name)
+		err = resourcedeploy.DeployCloudManagerSecret(newCluster)
+		if err != nil {
+			return fmt.Errorf("Unable to deploy cloud manager secret: %v", err)
+		}
+		crdStateStore := crd.NewCRDStore(&crd.CRDStoreOptions{
```

Comment:
This is not needed, along with `crdStateStore.Commit(newCluster)`. This is used by the `cluster-api` and is not needed in this case.